neo4j/conf.py

```diff
@@ -261,7 +261,10 @@ def get_ssl_context(self):
             ssl_context.check_hostname = False
             ssl_context.verify_mode = ssl.CERT_NONE     # https://docs.python.org/3.5/library/ssl.html#ssl.CERT_NONE
 
-        ssl_context.set_default_verify_paths()  # https://docs.python.org/3.5/library/ssl.html#ssl.SSLContext.set_default_verify_paths
+        # Must be load_default_certs, not set_default_verify_paths to work
+        # on Windows with system CAs.
```

Comment:
I think it worths mention the docs (https://docs.python.org/3.5/library/ssl.html#ssl.SSLContext.load_default_certs)

Reply:
Hmm, I was a bit annoyed over those references all over the code ;-) I mean finding info on standard python library functions isn't that hard to google